src/eval/execute.rs

```diff
@@ -178,6 +186,20 @@ fn generate_result_from_response(resp: Response, channel: Channel, is_private: b
     }
 }
 
+async fn paste_to_pb(client: &Client, code: &str) -> reqwest::Result<String> {
+    let resp = client
+        .post("https://paste.mozilla.org/api/")
+        .form(&[
+            ("lexer", "rust"),
+            ("expires", "2073600"),
+            ("content", code),
+        ])
+        .header("content-length", 0)
```

Comment:
I don't think this length is necessary. If this is required, I recommended you sending:

```rust
.header("content-length", code.as_bytes().len())
```

Reply:
oh. thx
it's required but not in use
I saw curl sends length of the entire request context, thus I was really confused

Reply:
Seems it's not a required field with correct request form